api/mapping/models.py

```diff
@@ -4,7 +4,10 @@
 from django.db import models
 from django.db.models.constraints import UniqueConstraint
 
-
+STATUS_CHOICES = [
+        ("Live", 'Live'),
```

Comment:
The first item in the tuple is usually uppercase, while the second is user friendly. 
("LIVE", "live").

Also, are Live and Archived the status options on Jenny's document?